homeassistant/components/aftership/sensor.py

```diff
@@ -30,6 +31,8 @@
 CONF_TRACKING_NUMBER = 'tracking_number'
 
 DEFAULT_NAME = 'aftership'
+DOMAIN = 'aftership'
```

Comment:
Let's call this something else than domain or don't define a constant at all. It's components that should define domains, not platforms.